extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitterConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.emitter.prometheus;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+/**
+ *
+ */
+public class PrometheusEmitterConfig
+{
+
+  @JsonProperty
+  private final String namespace;
+
+  @JsonProperty
+  private final String path;
+
+  @JsonCreator
+  public PrometheusEmitterConfig(
+      @JsonProperty("namespace") @Nullable String namespace,
+      @JsonProperty("path") @Nullable String path
+  )
+  {
+    this.namespace = namespace != null ? namespace : "druid";
```

Comment:
The default here is slightly hidden. Would this be better if set the default in the field itself, something like `@JsonProperty("namespace") @Nullable String namespace = "druid"` .